src/iort_lib/iort.cpp

```diff
@@ -16,11 +18,34 @@ using namespace std::chrono_literals;
 #include <deque>
 #endif
 
+std::string topic;
+struct mosquitto *mosq;
+bool new_msg = false;
+Json::Value payload;
```

Comment:
Quick question, does this work with multiple subscribers? I'm not sure how mosquitto works but I believe you may want to make these members of the Subscriber class instead of declaring them globally.